packages/syft/src/syft/lib/torch/tensor_util.py

```diff
@@ -128,3 +128,6 @@ def tensor_deserializer(buf: bytes) -> th.Tensor:
         return arrow_data_decoding(protobuf_tensor)
     elif protobuf_tensor.HasField("proto_data"):
         return protobuf_data_decoding(protobuf_tensor)
+
+    # TODO: Fix this
+    return protobuf_data_decoding(protobuf_tensor)
```

Comment:
this should be changed into `ValueError("Not protobuf decoding scheme set for Tensor.")`